static/css/cdr_common.css

```diff
@@ -195,14 +199,24 @@ li {
 }
 
 /** Header menu */
+
 #mainmenu {
-	margin: 0 0 0 0px;
+	margin: 0;
+    background-color: #E6E9EC;
+    padding: 0;
 }
 
-#mainmenu > li {
+#mainmenu > li
+/*, .submenu > li */
+{
 	float: left;
-	margin: 10px 0 0 0;
-	text-indent: 0px;
+	margin: 0;
+	text-indent: 0;
+    background-color: #E6E9EC;
```

Comment:
Please make sure you are using tab indents when editing these files.  There are a number of space indented new properties added